backend/src/main/java/edu/netcracker/backend/dao/mapper/TripMapper.java

```diff
@@ -0,0 +1,66 @@
+package edu.netcracker.backend.dao.mapper;
+
+import edu.netcracker.backend.model.Trip;
+import edu.netcracker.backend.model.User;
+import edu.netcracker.backend.model.state.trip.TripStateRegistry;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Component;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+@Component
+public class TripMapper implements RowMapper<Trip> {
+
+    private final TripStateRegistry tripStateRegistry;
+
+    @Autowired
+    public TripMapper(TripStateRegistry tripStateRegistry) {
+        this.tripStateRegistry = tripStateRegistry;
+    }
+
+    @Override
+    public Trip mapRow(ResultSet rs, int rowNum) throws SQLException {
```

Comment:
Operations with user repeat in this method. I suggest creating helper method in order to not to duplicate code

Reply:
Moved repeating logic to mapRow method.